src/types.ts

```diff
@@ -85,10 +92,26 @@ export interface ApiData {
   apiHost: string;
 }
 
+export type MockAppData = Omit<AppData, 'item' | 'creator' | 'member'> & {
+  itemId: string;
+  creatorId: string;
+  memberId: string;
+};
+export type MockAppAction = Omit<AppAction, 'item' | 'member'> & {
+  itemId: string;
+  memberId: string;
+};
+export type MockAppSetting = Omit<AppSetting, 'item' | 'creator'> & {
+  itemId: string;
+  creatorId: string;
+};
+
 export interface Database {
-  appData: AppData[];
-  appActions: AppAction[];
-  appSettings: AppSetting[];
-  members: Member[];
-  items: Item[];
+  appContext: LocalContext;
+  appData: MockAppData[];
+  appActions: MockAppAction[];
+  appSettings: MockAppSetting[];
+  // we pass the members as `CurrentMember` to be able to access all their properties, but they will be exposed as `Member`
```

Comment:
This is not compatible with mirage used in the current apps, but I think it's okay (not difficult to change)